### ISSUE TITLE
[2020]GraphNorm: A Principled Approach to Accelerating Graph Neural Network Training

### ISSUE THREAD

**Issue by ck-ksst** [ck-ksst/paper-reading#1]
# Abstract
**First**, they adapt and evaluate the existing methods from other domains to GNNs. 
Experiments with InstanceNorm was faster than ones with BatchNorm or LayerNorm. 
It was explained by showing that InstanceNorm serves as a preconditioner for GNNs, but such preconditioning effect is weaker with BatchNorm due to the heave batch noise in graph datasets. 
**Second** , GraphNorm can improve the generalization of GNNs, achieving better performance on graph classification benchmarks.

**Comment by ck-ksst** [comment]
# Motivation
The optimization of GNNs is less well understood, and in practice, the training of GNNs is often unstable and the convergence is slow. 
&rArr; They study how to improve the training of GNNs via normalization.
No single normalization helps in every domain and different architectures require specialized methods. 
&rArr; What normalization methods are effective for GNNs?

**Comment by ck-ksst** [comment]
# Method
